CHANGELOG.md

```diff
@@ -6,21 +6,18 @@ All user visible changes to this project will be documented in this file. This p
 
 
 
-## master
+## [0.11.0] · 2025-07-17
+[0.11.0]: https://github.com/instrumentisto/medea-jason/tree/medea-jason-0.11.0
 
-[Diff](https://github.com/instrumentisto/medea-jason/compare/medea-jason-0.10.0...master)
+[Diff](https://github.com/instrumentisto/medea-jason/compare/medea-jason-0.10.0...medea-jason-0.11.0)
 
-### BC Breaks
-
-- Library API:
-    - Renamed `ReconnectHandle` to `ReconnectHandleImpl` ([#223]).
```

Comment:
@turtletongue ,

Only core level was changed. I do not think, that we need to indicate it in the `CHANGELOG.md`, because external user uses this library with API level items, so I do not think, that it makes sense to write about internal changes here (because external user do not care about it).